vertex_ai/workbench_create_metadata/main.tf

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+# [START aiplatform_workbench_create_metadata]
+resource "google_workbench_instance" "default" {
+  name     = "workbench-instance-example"
+  location = "us-central1-a"
```

Comment:
These samples look good, just a question I had here: is "location" in this case the zone, as opposed to (on other resources I've seen) the region? The tf resource docs aren't clear on this separation. Will this be documented in the docs this sample will pair with?

Reply:
Resources are usually regional/zonal based on their end points. I don't think I've seen documentation mentioning this specifically anywhere. Usually this info is inferred from the values.